### ISSUE TITLE
The specification should be clear about the UCs and the scope

### ISSUE THREAD

**Issue by mounirlamouri** [sysapps/telephony#3]

**Comment by zolkis** [comment]
Use cases defined in http://www.w3.org/wiki/System_Applications_WG:_Telephony_API.
The API does specify the scope, but that has been challenged as too broad during the F2F.
The new proposed scope was cellular telephony (GSM, CDMA).

**Comment by zolkis** [comment]
Added clarifying comment to the Introduction section, with link to the use cases.

**Comment by mounirlamouri** [comment]
FTR, I think it would be better if we could get the use cases listed in the specification instead of pointing to a wiki page but this isn't terribly important I guess.

**Comment by marcoscaceres** [comment]
I agree, but we only really need to do that when we get to LC or CR (need to check the Process document): at some stage, we need to show the Director that the group's requirements for this specification have been met.

**Comment by marcoscaceres** [comment]
I've created a Last Call milestone and assigned this bug to it.
